src/main/resources/messages.properties

```
@@ -82,3 +82,10 @@ client.search.dob.hint = For example, 27/3/2007
 client.deceased.dod.label = When was your client's death registered?
 client.deceased.dod.hint = For example, 27/3/2007
 
+application.search.heading = Case and Application Search
+application.search.caption = Search for a case or application
+
+application.search.results.heading = Search Results
```

```suggestion
application.search.heading = Case and application search
application.search.caption = Search for a case or application

application.search.results.heading = Search results
```

Move towards sentence casing

---

Make sense, just followed the old UI.